authors/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get('DEBUG', 'True'),
 
-ALLOWED_HOSTS = ['localhost', '127.0.0.1','.herokuapp.com']
+ALLOWED_HOSTS = []
```

Comment:
Also, why do we need to change this, for this particular task?

Reply:
I hadn't noticed the hosts were removed. Let me revert back those changes.